presto-jdbc/src/main/java/com/facebook/presto/jdbc/PrestoConnection.java

```diff
@@ -642,10 +643,17 @@ private void setCatalogAndSchema()
         }
     }
 
+    @VisibleForTesting
```

Comment:
Nevermind, I see that this is more of an integration test that verifies the driver does the right thing with a `:443` in the URL. I will delete both these comments. (in case you read the first one via email)